README.md

```diff
@@ -108,6 +108,20 @@ persistedOperations?: { [hash: string]: string };
  * stores (e.g. S3).
  */
 persistedOperationsGetter?: PersistedOperationGetter;
+
+/**
+ * In development it's common to want to send arbitrary queries from GraphiQL
+ * whilst also enforcing Persisted Operations from the application. You should
+ * use this function if you want unpersisted operation to be allowed in some
+ * context (e.g in development).
+ * @example
+ * app.use(postgraphile(DATABASE_URL, SCHEMAS, {
+ *   allowUnpersistedOperation(req) {
+ *     return process.env.NODE_ENV === "development" && req.headers.referer.endsWith("/graphiql");
+ *   }
+ * });
+ */
+allowUnpersistedOperation?(request: any): boolean;
```

Comment:
I think the type for this is something like `IncomingMessage`? You'll also want the payload here:

```suggestion
allowUnpersistedOperation?(request: IncomingMessage, payload: any}): boolean;
```